scripts/globals/mobskills/whispers_of_ire.lua

```diff
@@ -0,0 +1,40 @@
+-----------------------------------
+-- Whispers of Ire
+-- Randomly absorbs 2 to 5 attributes from target.
+-- Type: Magical
+-- Notes: Can also use the ability to target itself and remove all debuffs (whispers_of_ire_self.lua)
+-----------------------------------
+require("scripts/globals/mobskills")
+require("scripts/globals/settings")
+require("scripts/globals/status")
+require("scripts/globals/msg")
+-----------------------------------
+local mobskill_object = {}
+
+mobskill_object.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskill_object.onMobWeaponSkill = function(target, mob, skill)
+    local amount = math.random(2, 5)
+    local count = 0
+
+    for i = 1, amount do
+        -- str down - chr down
+        local effectType = math.random(136, 142)
```

Comment:
We could make this a bit more human readable using the actual effect enum instead of raw numbers+comment.

Reply:
Could be this:
```suggestion
        local effectType = math.random(xi.effect.STR_DOWN, xi.effect.CHR_DOWN)
```

Should this be applying the same effect multiple times? If not then this should be randomly selecting from a list and removing them as they're selected.